microservices/nris_api/backend/app/nris/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from os.path import dirname, basename, isfile, join
+import glob
+modules = glob.glob(join(dirname(__file__), "*.py"))
+__all__ = [basename(f)[:-3] for f in modules if isfile(f) and not f.endswith('__init__.py')]
```

Comment:
what this do?

Reply:
:) I had actually just added comments for that. It scans for and imports all models, versus creating and maintaining hard references to every model.